packages/styles/src/themes/pentahoPlus.ts

```diff
@@ -363,6 +363,16 @@ const pentahoPlus = makeTheme((theme) => ({
     base: "6px",
   },
   components: {
+    HvBadge: {
+      classes: {
+        showCount: {
+          padding: "2px 5px",
```

Comment:
I don't think this change leads to the expected visual changes.
We already have this `2px` vertical padding, through `lineHeight` (it's `16px` instead of `12px`)